cmd/terramate/cli/cli.go

```diff
@@ -300,11 +389,25 @@ func (c *cli) run() error {
 
 func (c *cli) initStack(dirs []string) error {
 	var errmsgs []string
+
+	logger := log.With().
+		Str("action", "initStack()").
+		Logger()
+
+	logger.Debug().
+		Msg("Init stacks.")
 	for _, d := range dirs {
 		if !filepath.IsAbs(d) {
+			log.Trace().
+				Str("stack", fmt.Sprintf("%s%s", c.wd(), strings.Trim(d, "."))).
+				Msg("Make file path absolute.")
 			d = filepath.Join(c.wd(), d)
 		}
 
+		log.Debug().
+			Str("stack", fmt.Sprintf("%s%s", c.wd(), strings.Trim(d, "."))).
```

Comment:
hmm shouldn't we log the init parameters like c.root(), and the dir itself ("d") instead of this ? It also doesn't look at first glance as a proper stack path, stacks are relative to the project root (c.root()), not working dir (c.wd()). :thinking: